_data/library/jsfacades.yml

```diff
@@ -66,8 +66,20 @@
       dep: '"org.singlespaced" %%% "scalajs-d3" % "0.3.4"'
     - name: scaladget
```

Comment:
Each artifact should have its own name. The name is what goes in the first column of https://www.scala-js.org/libraries/facades.html